mat/cmatrix.go

```diff
@@ -32,8 +32,8 @@ type Conjugate struct {
 	CMatrix CMatrix
 }
 
-// At returns the value of the element at row i and column j of the transposed
-// matrix, that is, row j and column i of the Matrix field.
+// At returns the value of the element at row i and column j of the conjugate
+// transposed matrix, that is, row j and column i of the Matrix field.
 func (t Conjugate) At(i, j int) complex128 {
 	z := t.CMatrix.At(j, i)
 	return complex(real(z), -imag(z))
```

Comment:
At some point we are likely to be importing "math/cmplx". Use `cmplx.Conj`?

Reply:
Fixed the old code.